src/mame/quantel/dpb7000.cpp

```diff
@@ -1941,11 +2850,47 @@ void dpb7000_state::process_byte_from_disc(uint8_t data_byte)
 	}
 }
 
+bool dpb7000_state::is_disk_group_fdd(int group_index)
+{
+	static const uint16_t s_group_shifts[3] = { DGROUP_0_SHIFT, DGROUP_1_SHIFT, DGROUP_2_SHIFT };
+	uint16_t val = (m_config_sw34->read() >> s_group_shifts[group_index]) & 7;
+	return val == DGROUP_TYPE_FLOPPY;
+}
+
+bool dpb7000_state::is_disk_group_hdd(int group_index)
+{
+	static const uint16_t s_group_shifts[3] = { DGROUP_0_SHIFT, DGROUP_1_SHIFT, DGROUP_2_SHIFT };
+	uint16_t val = (m_config_sw34->read() >> s_group_shifts[group_index]) & 7;
+	return val == DGROUP_TYPE_80MB_CDC || val == DGROUP_TYPE_160MB_FUJITSU || val == DGROUP_TYPE_330MB_FUJITSU;
+}
+
+int dpb7000_state::get_heads_for_disk_group(int group_index)
+{
+	static const uint16_t s_group_shifts[3] = { DGROUP_0_SHIFT, DGROUP_1_SHIFT, DGROUP_2_SHIFT };
```

Comment:
Same for these – I’d prefer if the local constants used `constexpr` rather than `const` and used `SCREAMING_SNAKE_CASE` rather than an `s_` prefix.  Whether you make them `static` or not really doesn’t matter.

Reply:
Addressed as above.